src/Components/AdministratorPage/bidAudit/BidAuditCard.jsx

```diff
@@ -63,6 +63,34 @@ const BidAuditCard = ({ data, onEditModeSearch }) => {
     datePickerRef.current.setOpen(true);
   };
 
+  const submitRunAudit = () => {
+    dispatch(bidAuditRunAudit({
+      cycleId: cycle_id,
+      auditNbr: audit_id,
+    }));
+    swal.close();
+  };
+
+  const onRunBidAudit = () => {
+    swal({
+      title: 'Run this Bid Audit?',
+      button: false,
+      content: (
+        <div>
+          <button onClick={submitRunAudit} type="submit">Yes</button>
+          <button onClick={() => swal.close()}>Cancel</button>
```

Comment:
Anytime you have both a submit and cancel button next to each other I would keep in mind to differentiate the styling so that we prevent the accidental user clicks on submit vs cancel

```suggestion
          <button className="usa-button-secondary" onClick={() => swal.close()}>Cancel</button>
```